packages/splitter/README.md

```diff
@@ -24,26 +24,32 @@ The `useSplitter` hook manages positioning and required
 accessibility attributes for the window splitting separator.
 
 ```jsx static
-import { useSplitter, SplitterType, SplitterOrientation } from '@zendeskgarden/container-splitter';
+import {
+  useSplitter,
+  SplitterType,
+  SplitterOrientation,
+  SplitterPlacement
```

Comment:
Is this supposed to be `SplitterPosition`?

Reply:
Fixed in https://github.com/zendeskgarden/react-containers/pull/405/commits/72feeb0d9c03ec822bb561b57da6c3adc01a7996